src/bin/uhyve.rs

```diff
@@ -61,6 +61,14 @@ struct Args {
 	#[cfg(target_os = "linux")]
 	gdb_port: Option<u16>,
 
+	/// Paths that the kernel should be able to view, read or write.
+	///
+	/// Desired mount paths must be explicitly defined after a colon.
+	///
+	/// Example: --mount host_dir:guest_dir --mount file.txt:guest_file.txt
+	#[clap(long)]
+	mount: Vec<String>,
```

Comment:
We tend to let the user define parameters using the `UHYVE_<PARAMETER>` syntax. It was mentioned that "mount" is misleading here: https://github.com/hermit-os/uhyve/pull/783#discussion_r1856304860

However, there's an actual technical reason to adjust the name either on the Uhyve end or the Hermit end: `HERMIT_MOUNT` is used by the kernel to define the mountpoint used by Uhyve: https://github.com/hermit-os/uhyve/pull/783#issuecomment-2500108026

https://github.com/hermit-os/kernel/blob/c2c4b1f09df28566091173db5255b8e7e17a1a08/src/fs/uhyve.rs#L324

Reply:
opened this to remove at least one layer of confusion for starters https://github.com/hermit-os/kernel/pull/1477/

Reply:
`HERMIT_MOUNT` is a workaround for the old fs-access anyway.
I'd proceed here as follows:
- use `mount` in the arguments as you did
- let Uhyve set `HERMIT_MOUNT` to `/`
- remove `HERMIT_MOUNT` in the Kernel

Reply:
deemed as out of scope for this PR with the remap to `--file-mapping`, marking as resolved.